source/_components/light.mipow.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "Mipow Color"
+description: "Instructions how to setup Mipow color BT le bulb within Home Assistant."
+date: 2015-10-25
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mipow.png
+ha_category: Light
+ha_iot_class: "Local Polling"
+featured: false
+ha_release: xx
+---
+
+The `Mipow color` support is integrated into Home Assistant as a light platform. Please note that this will only work on linux and requires a bluetooth adapter with LE support!
```

Comment:
Linux  instead of Linux.

Suggest rewording this to
> Support for the `Mipow color` is integrated into Home Assistant as a light platform. Please note that this will only work on a Linux and requires Bluetooth LE support trough an USB adapter or a platform such as the Raspberry Pi 3 that has Bluetooth LE built in.